src/cli/arg/instrumentation.rs

```diff
@@ -33,6 +35,11 @@ pub struct Instrumentation {
     /// Which logger to use
     #[clap(long, env = "HARMONIC_LOGGER", default_value_t = Default::default(), global = true)]
     pub logger: Logger,
+    /// Tracing directives
+    ///
+    /// See https://docs.rs/tracing-subscriber/latest/tracing_subscriber/filter/struct.EnvFilter.html#directives
+    #[clap(long = "log-directive", global = true, env = "HARMONIC_LOG_DIRECTIVES", value_delimiter = ',', num_args = 0..)]
+    pub log_directives: Vec<Directive>,
```

Comment:
This basically duplicates `RUST_LOG` but allows the user to pass `--log-directive harmonic=trace` instead of needing to set it. This is mostly useful for CI as I don't imagine many people will use trace logging.